torch/onnx/_internal/fx/export_options.py

```diff
@@ -0,0 +1,33 @@
+"""Options for FX exporter static kwargs."""
+from __future__ import annotations
+
```

Comment:
"export_options.py" -> "options.py"?

Reply:
Tried to move verification options inthis too, but failed due to https://github.com/pytorch/pytorch/issues/94800

Reply:
Actually ... I meant file renaming lol. Sry

Reply:
I am moving VerificationOptions also into options.py (and move whole thing outside of fx, but under _internal.), and it turns out the name options.py is too general, as most of our variables/parameters are already named options. So I will just go with kwarg_options,py. What do you think?